src/createServer.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const cors = require('cors');
+const { authRouter } = require('./routers/auth.router');
+const { errorMiddleware } = require('./middlewares/error.middleware');
+const cookieParser = require('cookie-parser');
+const { profileRouter } = require('./routers/profile.router');
+const { authMiddleware } = require('./middlewares/auth.middleware');
+
+function createServer() {
+  const app = express();
+
+  app.use(cors({ origin: '*', credentials: true }));
```

Comment:
The CORS configuration allows requests from any origin ('*'). This can be a security risk, especially if sensitive data is involved. Consider restricting the allowed origins to specific domains that should have access to your server.